### PR TITLE
Adjust Identifier properties

### PR DESCRIPTION
**Pull request by jnyrup**
Inspired by #1772 I looked over the other `Identifier` properties to align them better.

### CODE REVIEW

File: Src/FluentAssertions/Types/MethodBaseAssertions.cs [hunk 1]
```diff
@@ -93,8 +93,6 @@ public AndConstraint<TAssertions> NotHaveAccessModifier(CSharpAccessModifier acc
             return new AndConstraint<TAssertions>((TAssertions)this);
         }
 
-        protected override string Identifier => "methodBase";
```
Comment (jnyrup) [inline Src/FluentAssertions/Types/MethodBaseAssertions.cs:96]: `MethodBaseAssertions` is `abstract` and its two derivatives `ConstructorInfoAssertions` and `MethodInfoAssertions` overrides `Identifier` with `"constructor"` and `"method"`, respectively.

**Review by dennisdoomen — APPROVED** [review]
Maybe we should also improve the documentation on the base-class property to emphasize it should be a user-friendly name.

**Comment by jnyrup** [comment]
> Maybe we should also improve the documentation on the base-class property to emphasize it should be a user-friendly name.

Like this?
image

**Comment by dennisdoomen** [comment]
> Like this?

Exactly

**Pull request merged**